src/content/releases/_index.en.md

```diff
@@ -5,6 +5,20 @@ pre = "<b>6. </b>"
 weight = 30
 +++
 
+## v0.7.0 StatefulSet support for service discovery and benchmark tooling
+
+> This release mainly focused on adding support for StatefulSets in Lighthouse for service discovery and adding new `subctl`
+> commands to benchmark the network performance across clusters.
+
+* **Lighthouse** enhancements/changes:
+  * Added support for accessing individual pods in a StatefulSet using their host names.
+  * A service in a specific cluster can now be explicitly queried.
+  * Removed support for the **supercluster**.local domain to align with the Kubernetes MultiCluster Service API.
+* Added new `subctl` benchmark commands for measuring the throughput and round trip latency between two Pods in separate
+  clusters or within the same cluster.
+* The datapath is no longer disrupted when the **Globalnet** Pod is restarted.
+* The Route Agent component now runs on all worker nodes include those with taints.
```

Comment:
@nyechiel @mangelajo Shall we capture that we did some eBPF/OVN analysis and work on route-agent pluggable drivers is in progress?

Reply:
IMO release notes capture what’s new in the release, not what happened (or what’s planned) in the project. So if the code in the release changed to start supporting route-agent pluggable drivers, that can go in the release notes, but otherwise I’d avoid mentioning this (not to discount the effort involved of course!).

Reply:
Makes sense

Reply:
That was my thought too.